### PR TITLE
Show message on succesful registration.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@comzeradd r?

**Comment by akatsoulas** [comment]
r+

**Comment by johngian** [comment]
@ppapadeas Can you r? the text?

**Comment by comzeradd** [comment]
Since we call this settings now, I'd suggest we display something like this: "Your registration is complete. Head over to Settings for more options". Or something like that. It would probably make sense to linkify the `Settings`.

**Comment by johngian** [comment]
@akatsoulas r?

**Comment by ppapadeas** [comment]
Text should be:

"Your registration is complete. Feel free to visit the Settings page to complete additional info for your profile."

With a link on the word "Settings"

**Comment by johngian** [comment]
@akatsoulas PR updated

File: mozillians/phonebook/views.py [hunk 1]
```diff
@@ -242,10 +243,16 @@ def get_request_data(form):
                 f.save()
 
             if curr_sect == 'registration_section':
+                settings_url = reverse('phonebook:profile_edit')
+                settings_link = '<a href="{}">settings</a>'.format(settings_url)
```
Comment (akatsoulas) [inline mozillians/phonebook/views.py]: User a var for readability, something like `settings_url`.

Comment (akatsoulas) [inline mozillians/phonebook/views.py]: You could also user positional arguments if you like.

File: mozillians/templates/base.html [hunk 2]
```diff
@@ -163,7 +163,7 @@
             {% for message in messages %}
               {% with label = message.tags in ['error', 'success', 'info', 'warning'] %}
                 <div class="alert {% if label %}alert-{{ message.tags }}{% endif %}">
-                  {{ message }}
+                  {{ message.message }}
```
Comment (akatsoulas) [inline mozillians/templates/base.html]: Can we do something better than this?

**Comment by akatsoulas** [comment]
A small test, looking for the error would be really nice :)

**Comment by akatsoulas** [comment]
r+wc :rocket:

**Pull request merged**